modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -275,41 +267,28 @@ class ViewDetails extends \NDB_Form
     /**
      * Compares parameters of a series and of the protocols
      *
-     * @param string $seriesRange   series parameter to be compared
-     * @param string $protocolRange protocol's parameter
+     * @param string $seriesRange series parameter to be compared
+     * @param string $protocolMin minimum value of protocol's parameter
      *        to be compared with
+     * @param string $protocolMax maximum value of protocol's parameter
+     *         to be compared with
      *
      * @return bool true if series parameter is in range with
      *         the protocol's one, false otherwise
      */
-    function _inRange($seriesRange, $protocolRange)
+    function _inRange($seriesRange, $protocolMin, $protocolMax)
     {
-        if (! isset($seriesRange)) {
-            return false;
-        } else if (! isset($protocolRange)) {
+        if (!isset($seriesRange)) {
```

Comment:
@llevitis I think here you want to return true now that you don't check anymore above for the existence of that value before calling `inRange`. That fixed the bug on my sandbox.